git-helper.js

```diff
@@ -34,19 +34,19 @@ exec('git tag --list', options)
 
 // Tags the repository at the current state
 const tag = (tag, message, push, options) =>
-exec(`git tag -a ${tag} -m "${message}"`, options)
+exec(`git tag -a ${tag} -m "${message}" ${options.branch || ''}`, options)
```

Comment:
Don't set branch here as it would be confusing not using the current local state of the repository.